src/tests/stdlib.rs

```diff
@@ -17,11 +17,32 @@ fn stdlib_test(input: &str) {
     let code = fs::read_to_string(input)
     .expect(&format!("Failed to open {input} test file"));
 
-    let output = match Path::new(&input.replace(".ab", ".output.txt")).exists() {
-        true => fs::read_to_string(input.replace(".ab", ".output.txt"))
-        .expect(&format!("Failed to open {input}.output.txt file")),
-        _ => "Succeded".to_string()
-    };
+    let mut _is_output = false;
```

Comment:
@Mte90 why are we using `_` before the variable name if we use the variable?

Reply:
in my mind it is just a temporary variable not so much important, it is part of my codestyle in that way.

Reply:
@Mte90 I understand. Can we stick to the style in which the rest of the codebase is written? I think that if there is too much byproduct variables (the temporary variables) then it's a sign that this variable has to be split into smaller functions.